### ISSUE TITLE
Youtube Ads Not Bring Blocked By Brave Ad Blocker

### ISSUE THREAD

**Issue by SonicRoshan** [qutebrowser/qutebrowser#6578]
**Version info**:
qutebrowser v2.3.0
Git commit: 
Backend: QtWebEngine 5.15.5, based on Chromium 87.0.4280.144
Qt: 5.15.2

**Does the bug happen if you start with `--temp-basedir`?**:
Yes

**Description**
Qutebrowser used to block youtube ads, but now it does not work.

**How to reproduce**
Install ```python-adblock``` using AUR
Open youtube

**Comment by The-Compiler** [comment]
Duplicate of #6480

**Comment by SonicRoshan** [comment]
Sorry for not seeing that, so as of now qutebrowser does not support proper youtube ad blocking?

**Comment by The-Compiler** [comment]
Someone on Reddit [recently wrote a Greasemonkey script](https://old.reddit.com/r/qutebrowser/comments/ntl2ko/easy_youtube_adblocker_greasemonkey_script/h15yaqc?utm_source=share&utm_medium=web2x&context=3) to do so. I haven't tested it myself yet. Another approach is to use an external player like `mpv` instead, see 10. in the [FAQ](https://github.com/qutebrowser/qutebrowser/blob/master/doc/faq.asciidoc).

**Comment by SonicRoshan** [comment]
Hey, I found a way to completely remove youtube ads. Just add contents of this file to /etc/hosts and restart dns cache https://ewpratten.retrylife.ca/youtube_ad_blocklist/hosts.ipv4.txt
